services/QuillLMS/engines/evidence/app/workers/evidence/research/gen_ai/build_llm_example_worker.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Evidence
+  module Research
+    module GenAI
+      class BuildLLMExampleWorker
+        include Evidence.sidekiq_module
+
+        sidekiq_options retry: 0, queue: 'low'
```

Comment:
I think we want these in the `default` queue. if we are considering staff to be "users" who are waiting on info in the UI, you could even make a case for `critical_external`, but I think since it's staff, `default` is a nice middle ground.